recipes/takeabreak/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+mkdir -p ${PREFIX}/bin
+
+# decrease RAM needed
+sed -i.bak 's/make -j4/make -j1/' INSTALL
+sed -i.bak 's/.\/build\/bin\/TakeABreak/.\/build\/bin\/TakeABreak -max-memory 1000/' INSTALL
```

Comment:
why is this may-memory needed?

Reply:
Is travis not handling this well?

Reply:
I hope so. They are limited, but we never had problems before.

Reply:
Ohoho. I have to increase my virtual machine for my developments :).